src/app/commands/cmd_change_pixel_format.cpp

```diff
@@ -460,163 +461,82 @@ class ColorModeWindow : public app::gen::ColorMode {
 
 } // anonymous namespace
 
-class ChangePixelFormatCommand : public Command {
+struct ChangePixelFormatParams : public NewParams {
+  Param<bool> ui{ this, false, "ui" };
+  Param<PixelFormat> format{ this, IMAGE_RGB, "format" };
+  Param<render::DitheringAlgorithm> dithering{ this,
+                                               render::DitheringAlgorithm::None,
+                                               "dithering" };
+  Param<std::string> matrix{ this, "", "dithering-matrix" };
+  Param<double> factor{ this, 1.0, "dithering-factor" };
```

Comment:
Add param aliases (these are easier to use in Lua):
```suggestion
  Param<std::string> matrix{ this, "", { "ditheringMatrix", "dithering-matrix" } };
  Param<double> factor{ this, 1.0, { "ditheringFactor", "dithering-factor" } };
```